ring-core/src/ring/util/response.clj

```diff
@@ -44,6 +44,14 @@
       :headers {"Location" url}
       :body    body}))
 
+(defn bad-request
+  "Returns a Ring response for a HTTP 201 created response."
+  {:added "1.4"}
```

Comment:
Incorrect "added" metadata. This should be 1.7.

Reply:
Fixed!